php/libraries/Utility.class.inc

```diff
@@ -1123,7 +893,10 @@ class Utility
      *               array(0 => array('Test_name_display' => $TestName))?
      *
      * @note    Function comment written by Dave, not the author of this function.
-     *       The return value format seems weird. Where is this used?
+     *       The return value format seems weird. Should possibly be refactored.
+     * @note    Used in:
+     *      - data_team_helper/php/NDB_Form_data_team_helper.class.inc
+     *      - data_team_helper/ajax/GetInstruments.php
```

Comment:
A redmine ticket saying to move this to the data_team_helper and verify that that's the only place that it's used is better than a comment saying where it's used (which would need to be maintained, and may become outdated..)